testsuite/forge/src/backend/k8s/mod.rs

```diff
@@ -174,7 +180,29 @@ impl Factory for K8sFactory {
                     uninstall_testnet_resources(self.kube_namespace.clone()).await?;
                     bail!(e);
                 },
+            };
+
+            // add an indexer too!
+            if self.enable_indexer {
+                // NOTE: by default, use a deploy profile and no additional configuration values
+                let values = ForgeDeployerValues {
+                    profile: DEFAULT_FORGE_DEPLOYER_PROFILE.to_string(),
+                    era: new_era.clone().expect("Era not set in created testnet"),
+                    namespace: self.kube_namespace.clone(),
+                    indexer_grpc_values: None,
+                    indexer_processor_values: None,
+                };
+
+                let forge_deployer_manager =
```

Comment:
A couple of thoughts on ForgeDeployerManager (sorry I know not implemented here but I wanted to share since we're actuvely using it)

1) ForgeDeployerManager doesn't actually need a type, it would make more sense to have that built in to the ForgeDeployerValues and either have an enum or a grpc that could handle the serialization / deserialization or just have a less strongly typed map of values. Consider either
```
struct ForgeBackendConfig {
  namespace: String, // Namespace to run in? Not sure if this even should be a property you specify, should forge decide this for you?
  image: String, // Full qualifier of the backend image
  values:  serde_json::Value,
}
```

Because in internal-ops you're deserializing this, you either want to have no strict serialization (since you already just render this to json) or you should use grpc. Right now you kind of have the restriction of types without the actual enforcement and the downside of backwards compat issues if you change the struct on either side (grpc would allow to handle that more safely)